sources/packages/backend/apps/api/src/route-controllers/program-year/program-year.controller.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { ProgramYearService } from "../../services";
+import { OptionItemAPIOutDTO } from "../models/common.dto";
+
+@Injectable()
+export class ProgramYearControllerService {
+  constructor(private readonly programYearService: ProgramYearService) {}
+
+  async getProgramYears(): Promise<OptionItemAPIOutDTO[]> {
+    const programYears = await this.programYearService.getProgramYears();
+    if (!programYears) {
```

Comment:
Probably something from the past but we do not thow `NotFoundException` for an empty list.
The list can just be returned as empty.